api/v1alpha1/physical_machine_chaos_webhook.go

```diff
@@ -131,8 +133,6 @@ func (in *PhysicalMachineChaosSpec) Validate(root interface{}, path *field.Path)
 	case PMClockAction:
 		validateConfigErr = validateClockAction(in.Clock)
 	default:
-		allErrs = append(allErrs,
```

Comment:
action already checked in https://github.com/chaos-mesh/chaos-mesh/blob/80c4d00a302de1f1d2dca609395b05d2c4db1fdc/api/v1alpha1/physical_machine_chaos_types.go#L67, so removed it